code/datums/mutations/actions.dm

```diff
@@ -230,3 +230,140 @@
 
 	var/obj/item/bodypart/BP = pick(parts)
 	BP.dismember()
+
+/datum/mutation/human/tongue_spike
+	name = "Tongue Spike"
+	desc = "Allows a creature to voluntary shoot their tongue out...?"
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>Your feel like you can throw your voice.</span>"
+	instability = 15
+	power = /obj/effect/proc_holder/spell/self/tongue_spike
+
+	energy_coeff = 1
+	synchronizer_coeff = 1
+
+/obj/effect/proc_holder/spell/self/tongue_spike
+	name = "Launch spike"
+	desc = "Shoot your tongue out in the direction you're facing, embedding it for a fairly large amount of damage."
+	clothes_req = FALSE
+	human_req = FALSE
+	charge_max = 100
+	action_icon = 'icons/mob/actions/actions_genetic.dmi'
+	action_icon_state = "spike"
+	var/spike_path = /obj/item/hardened_spike
+
+/obj/effect/proc_holder/spell/self/tongue_spike/cast(mob/user = usr)
+	if(!iscarbon(user))
+		return
+
+	var/mob/living/carbon/C = user
+	if(HAS_TRAIT(C, TRAIT_NODISMEMBER))
+		return
+	var/obj/item/organ/tongue/tongue
+	for(var/org in C.internal_organs)
+		if(istype(org, /obj/item/organ/tongue))
+			tongue = org
+			break
+
+	if(!tongue)
+		to_chat(C, "<span class='notice'>You don't have a tongue to shoot!</span>")
+		return
+
+	tongue.Remove(C, special = TRUE)
+	var/obj/item/hardened_spike/spike = new spike_path(get_turf(C), C)
+	tongue.forceMove(spike)
+	spike.throw_at(get_edge_target_turf(C,C.dir), 14, 4, C)
+
+/obj/item/hardened_spike
+	name = "biomass spike"
+	desc = "Hardened biomass, shaped into a spike. Very pointy!"
+	icon_state = "tonguespike"
+	force = 2
+	throwforce = 15 //15 + 2 (WEIGHT_CLASS_SMALL) * 4 (EMBEDDED_IMPACT_PAIN_MULTIPLIER) = i didnt do the math
```

Comment:
that's hardly a "fairly large chunk of damage"...

Reply:
pain damage on shurikens are crazy just take my word on this, this 15 damage procs over and over and over